### PR TITLE
more style updates

### PR DESCRIPTION
**Pull request by angelanicholas**
- removes .sass-cache files (already added to .gitignore earlier but forgot to delete these)
- updates button styles:

screen shot 2018-03-08 at 10 01 58 am

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -19,11 +19,9 @@ Specifically, our supporters have provided:
 
 **This directory is open-source.** Any firm can add and update their policies and point(s)-of-contact, and we hope that every major venture firm will participate.
 
-<div align="center">
-<button>
-<a href="https://airtable.com/shrudsWooa1wa7sCm">Join #MovingForward</a>
-</button>
-</div>
+<p style="text-align:center">
```
Comment (triketora) [inline index.md]: Preferable to not inline styles in HTML - can you put a classname on the `<p>` and style it from CSS?

**Comment by triketora** [comment]
Thank you for removing the sass cache!!

**Review by triketora — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (angelanicholas) [reply index.md]: yea sure, was just going off what was there before but happy to change it one sec

**Review by triketora — APPROVED** [review]

**Comment by triketora** [comment]
Yay!

**Pull request merged**